src/i18n/es.js

```diff
@@ -34,12 +34,20 @@ const urlMasterSubscriber = `${urlDopplerLegacy}/Lists/MasterSubscriber/`;
 
 export default {
   change_plan: {
+    all_of_plan: `Todo lo de {plan} más:`,
     calculate_price: 'Calcular Precio',
+    cancel_campaign: 'Cancelar Campaña',
+    compare_features: 'Comparar Funcionalidades',
     description: '¿Realizas envíos periodicamente o un newsletter al mes?',
+    email_parameter: 'Parámetros de Email',
+    features: 'Funcionalidades:',
     per_month: 'por mes',
     recommended: 'Recomendado',
     selected_type: 'Planes por: ',
+    shipping_limit: 'Límite de envíos',
```

Comment:
```suggestion
    shipping_limit: 'Límite de Envíos',
```
This is for consistency with English version, nothing more.